src/main/java/robot/subsystems/elevator/commands/ElevatorCommand.java

```diff
@@ -0,0 +1,60 @@
+package robot.subsystems.elevator.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+import robot.subsystems.elevator.Constants;
+import robot.subsystems.elevator.Elevator;
+
+/**
+ *  Move the elevator to a certain height
+ */
+public class ElevatorCommand extends Command {
+    private double delta = 0.1;
+    private Elevator elevator = Robot.elevator;
```

Comment:
Do not do this. Aliasing is not good because you might use one variable but think you're using the other one.